### PR TITLE
Flake8 CI fixes

### PR DESCRIPTION
**Pull request by Zen-CODE**
A few notes on this PR:
* it favours adding inline exceptions to violations rather that adding permanent rules
* it prevent flake 8 on certain files to prevent clashes with this PR: https://github.com/kivy/kivy-ios/pull/443. Once these two PRs are merged, the file exceptions in `tox.ini` will be removed in another PR.

### CODE REVIEW

File: recipes/python2/__init__.py [hunk 1]
```diff
@@ -130,7 +133,7 @@ def reduce_python(self):
             os.chdir(join(self.ctx.dist_dir, "root", "python2", "lib", "python2.7"))
             sh.find(".", "-iname", "*.pyc", "-exec", "rm", "{}", ";")
             sh.find(".", "-iname", "*.py", "-exec", "rm", "{}", ";")
-            #sh.find(".", "-iname", "test*", "-exec", "rm", "-rf", "{}", ";")
+            # sh.find(".", "-iname", "test*", "-exec", "rm", "-rf", "{}", ";")
```
Comment (AndreMiras) [inline recipes/python2/__init__.py:136]: I would actually delete that line, but it doesn't matter that much

File: recipes/pkgresources/pkg_resources.py [hunk 2]
```diff
@@ -617,9 +642,9 @@ def resolve(self, requirements, env=None, installer=None,
                             ws = WorkingSet([])
                     dist = best[req.key] = env.best_match(req, ws, installer)
                     if dist is None:
-                        #msg = ("The '%s' distribution was not found on this "
+                        # msg = ("The '%s' distribution was not found on this "
                         #       "system, and is required by this application.")
-                        #raise DistributionNotFound(msg % req)
+                        # raise DistributionNotFound(msg % req)
```
Comment (AndreMiras) [inline recipes/pkgresources/pkg_resources.py:647]: We might as well just remove dead code

File: .ci/rebuild_updated_recipes.py [hunk 3]
```diff
@@ -16,12 +16,14 @@ def modified_recipes(branch='origin/master'):
     for file_path in git_diff:
         if 'recipes/' in file_path:
             recipe = file_path.split('/')[1]
-            recipes.add(recipe)
+            if recipe not in ["python", "hostpython"]:  # alias recipes
+                recipes.add(recipe)
```
Comment (AndreMiras) [inline .ci/rebuild_updated_recipes.py:20]: did you sneak a bugfix in this linter PR?

**Comment by AndreMiras** [comment]
Didn't we just break the CI?
```
[ERROR   ] No recipe named python.py
/bin/sh: line 1: pycrypto: command not found
/bin/sh: line 2: click: command not found
Traceback (most recent call last):
  File ".ci/rebuild_updated_recipes.py", line 27, in <module>
    subprocess.run(f"python3 toolchain.py build {updated_recipes}", shell=True, check=True)
  File "/Users/runner/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/subprocess.py", line 512, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command 'python3 toolchain.py build curly ios werkzeug libcurl host_setuptools freetype photolibrary markupsafe distribute jinja2 libzbar python.py
 pycrypto flask ffpyplayer pyobjus hostpython.py
 click numpy audiostream kivent_core libjpeg hostpython2 sdl2_mixer python2 sdl2_ttf openssl pyyaml zbarlight python3 pkgresources cymunk libpng ffmpeg pillow plyer hostlibffi pil itsdangerous' returned non-zero exit status 127.
```

File: .ci/rebuild_updated_recipes.py (hunk 3 above)
Comment (Zen-CODE) [reply .ci/rebuild_updated_recipes.py:20]: Well, they  seem redundant? They are alias recipes for `python2` or `python3' and `hostpyhton2` and `hostpython3` respectively. So the actual recipes are checked, it 's just the alias that does not exists. From what I understand anyway.

Comment (AndreMiras) [reply .ci/rebuild_updated_recipes.py:20]: I haven't dug into it, but the CI is young enough and we would take responsibility to revert or fix if we introduced a bug here. So I'm OK that we take the risk

File: .ci/rebuild_updated_recipes.py [hunk 4]
```diff
@@ -16,12 +16,14 @@ def modified_recipes(branch='origin/master'):
     for file_path in git_diff:
         if 'recipes/' in file_path:
             recipe = file_path.split('/')[1]
-            recipes.add(recipe)
+            if recipe not in ["python.py", "hostpython.py"]:  # alias recipes
+                recipes.add(recipe)
```
Comment (AndreMiras) [inline .ci/rebuild_updated_recipes.py:20]: Actually now that I think of it, we're already using a set so this is dead code in a way

File: .ci/rebuild_updated_recipes.py (hunk 3 above)
Comment (AndreMiras) [reply .ci/rebuild_updated_recipes.py]: BTW touching it I understood what you were doing. I misread this:
```
if recipe not in ["python", "hostpython"]:
```
For that:
```
if recipe in recipes:
```
Sorry about that :facepalm:

**Pull request merged**